liTest run scala.scalanative.junit.ExceptionBeforeClass started
leTest scala.scalanative.junit.ExceptionBeforeClass failed: java.lang.IllegalArgumentException: foo, took <TIME>
e2scala.scalanative.junit.ExceptionBeforeClass
liTest run scala.scalanative.junit.ExceptionBeforeClass finished: 1 failed, 0 ignored, 0 total, <TIME>
d
